static/src/js/components/MetadataForm/MetadataForm.jsx

```diff
@@ -138,6 +140,18 @@ const MetadataForm = () => {
 
   useEffect(() => {
     const { draft: fetchedDraft } = data || {}
+    const { revisionId: cmrRetrievedRevisionId } = fetchedDraft || ''
+
+    if (cmrRetrievedRevisionId && (cmrRetrievedRevisionId !== appContextRevisionId)) {
```

Comment:
I can't remember why you needed to verify it wasn't null.   I'm curious why you are only checking if cmrRetrievedRevisionId is null, we don't need to check if both non null?

Reply:
You're right I needed to add appContextRevisionId is undefined as well. Both these will be undefined upon clicking +New Draft. AppcontextRevisionId will be undefined upon refresh.